### PR TITLE
Sabsefix

### PR DESCRIPTION
**Pull request by chappy2**
new pr for new commit

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/juanmi108/Super-Global-Hyper-Mega-Net-nosql/pull/44" target="_blank">ReviewNinja

**Comment by chappy2** [comment]
#40 Change

File: routes/list.js [hunk 1]
```diff
@@ -138,6 +138,7 @@ router.post('/:id', function(req, res, next) {
 router.post('/', function(req, res, next) {
 
     var newList = new listModel(req.listDoc);
+    newList.owner= req.user._name;
```
Comment (juanmi108) [inline routes/list.js]: the user is set at the beginning of the file. no need to rewrite it here. !fix

Comment (chappy2) [inline routes/list.js]: I didn't See IT. I fix IT later

**Pull request merged**